tests/core/creators/manual/resizewithcssunits.md

```diff
@@ -0,0 +1,15 @@
+@bender-tags: bug, 4.14.1, 1883
+@bender-ui: collapsed
+@bender-ckeditor-plugins: toolbar, wysiwygarea, table
+
+1. Open browser developer tools.
+
+1. In console paste: `CKEDITOR.instances[ 'editor' ].resize( '20em', '10em' )`
```

Comment:
Using developer tools to test this issue is not the most optimal approach as we cannot test it on mobiles. It also requires additional test steps. Maybe we could go with more complex test e.g. by adding CSS style inputs?
![image](https://user-images.githubusercontent.com/10279586/80952041-4f001500-8df9-11ea-99aa-693f33342849.png)